atomic_reactor/inner.py

```diff
@@ -345,6 +345,14 @@ class ImageBuildWorkflowData(ISerializer):
     image_id: Optional[str] = None
     parent_images_digests: Dict[str, Dict[str, str]] = field(default_factory=dict)
 
+    # List of output files that are uploaded to Brew/Koji
+    # Each element is a two-strings list, local_filename and dest_filename. E.g.
+    # [
+    #     {"local_filename": "/path/to/data.json", "dest_filename": "metadata.json"},
+    #     {"local_filename": "/path/to/build.log", "dest_filename": "x86_64-build.log"},
+    # ]
+    koji_upload_files: List[Dict[str, str]] = field(default_factory=list)
```

Comment:
This could also be a good way to save logs from `bsi` (source) / `podman` (binary) as a replacement for the removed BuildResult.logs :+1: